django/core/mail/utils.py

```diff
@@ -18,3 +21,24 @@ def get_fqdn(self):
 
 
 DNS_NAME = CachedDnsName()
+
+
+def check_mail_setting(value):
+    """
+    Check each item in the value to be a tuple of (Full name, email address)
+    """
+    if not value:
+        return False
+
+    for item in value:
+        if not isinstance(item, tuple) or len(item) != 2:
+            return False
+
+        name, email = item
+        if not isinstance(name, str) or not name.strip():
+            return False
+        try:
+            validate_email(email)
+        except ValidationError:
+            return False
```

Comment:
I don't think that we should validate emails addresses here because e.g. app can use local-only addresses for managers/admins emails e.g. `app-admin` (see 1564e42ad397021093585147875a21dae1a3b3fc).